include/libpfc.h

```diff
@@ -121,6 +128,10 @@ _pfc_asm_code_(op)                              \
 
 void      pfcRemoveBias     (PFC_CNT* b, int64_t mul);
 
+/**
+ * Return a string representation of an error code such as those returned by pfcInit
+ */
+const char *pfcErrorString(int err);
```

Comment:
In several places I added some error handling, and some methods now return a specific error code. You can pass the code in there to get an error string, which I use for display. If the error code isn't known (e.g., for some methods that still return 1), you just get "Unknown error".